src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -0,0 +1,428 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { LitElement, html, customElement, property } from 'lit-element';
+import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+
+import { Providers } from '../../Providers';
+import { ProviderState } from '../../providers/IProvider';
+import { styles } from './mgt-people-picker-css';
+
+import '../mgt-person/mgt-person';
+import '../../styles/fabric-icon-font';
+import { MgtTemplatedComponent } from '../templatedComponent';
+import { MgtPersonDetails, MgtPerson } from '../mgt-person/mgt-person';
+
+@customElement('mgt-people-picker')
+export class MgtPicker extends MgtTemplatedComponent {
+  private _firstUpdated = false;
+  private groupPeople;
+  private isLoading = false;
+
+  @property({
+    attribute: 'people',
+    type: Object
+  })
+  people: Array<MgtPersonDetails> = null;
+
+  @property({
+    attribute: 'show-max',
+    type: Number
+  })
+  showMax: number = 6;
+
+  private arrowSelectionCount: number = 0;
+
+  @property({
+    attribute: 'group',
+    type: String
+  })
+  group: string;
+
+  @property() private _personName: string = '';
+  @property() private _selectedPeople: Array<any> = [];
```

Comment:
Make public without underscore

Reply:
Updated to public
https://github.com/microsoftgraph/microsoft-graph-toolkit/commit/dc8dca87b40f7fce4c6f67a710ad9d3bccae3004